ReactWindows/ReactNative/ReactInstanceManagerImpl.cs

```diff
@@ -0,0 +1,238 @@
+                    nativeRegistryBuilder,
+                    jsModulesBuilder);
+            }
+
+            foreach (var reactPackage in _packages)
+            {
+                using (Tracer.Trace(Tracer.TRACE_TAG_REACT_BRIDGE, "createAndProcessCustomReactPackage"))
+                {
+                    ProcessPackage(
+                        reactPackage,
+                        reactContext,
+                        nativeRegistryBuilder,
+                        jsModulesBuilder);
+                }
+            }
+
+            var nativeModuleRegistry = default(NativeModuleRegistry);
+            using (Tracer.Trace(Tracer.TRACE_TAG_REACT_BRIDGE, "buildNativeModuleRegistry"))
+            {
+                nativeModuleRegistry = nativeRegistryBuilder.Build();
+            }
+
+            var javaScriptModulesConfig = default(JavaScriptModulesConfig);
+            using (Tracer.Trace(Tracer.TRACE_TAG_REACT_BRIDGE, "buildJSModuleConfig"))
+            {
+                javaScriptModulesConfig = jsModulesBuilder.Build();
+            }
+
+            var javascriptRuntime = new CatalystInstance.Builder
+            {
+                QueueConfigurationSpec = CatalystQueueConfigurationSpec.Default,
+                JavaScriptExecutor = jsExecutor,
+                Registry = nativeModuleRegistry,
+                JavaScriptModulesConfig = javaScriptModulesConfig,
+                BundleLoader = jsBundleLoader,
+                NativeModuleCallExceptionHandler = ex => { } /* TODO */,
+            }.Build();
+
+            reactContext.InitializeWithInstance(javascriptRuntime);
+
+            await javascriptRuntime.InitializeBridgeAsync();
+            
+            return var;
+        }
+
+        private void ProcessPackage(
+            IReactPackage reactPackage,
+            ReactApplicationContext reactContext,
+            NativeModuleRegistry.Builder nativeRegistryBuilder,
+            JavaScriptModulesConfig.Builder jsModulesBuilder)
+        {
+            foreach (var nativeModule in reactPackage.CreateNativeModules(reactContext))
+            {
+                nativeRegistryBuilder.Add(nativeModule);
+            }
+
+            foreach (var type in reactPackage.CreateJavaScriptModulesConfig())
+            {
+                jsModulesBuilder.Add(type);
+            }
+        }
+
+        /// <summary>
+        /// Attaches the <see cref="ReactRootView" /> to the list of tracked root views
+        /// </summary>
+        /// <param name="rootView">The root view for the ReactJS app</param>
+        public override void AttachMeasuredRootView(ReactRootView rootView)
+        {
+            _attachedRootViews.Add(rootView);
+            
+            if (var != null)
+            {
+                AttachMeasuredRootViewToInstance(rootView, var.CatalystInstance);
+            }
+        }
+
+        /// <summary>
+        /// Detach given <see cref="rootView" /> from current catalyst instance. 
```

Comment:
I think there might be a tag called <paramref> that you should use here instead.